webview-ui/src/components/chat/checkpoints/CheckpointSaved.tsx

```diff
@@ -8,7 +8,7 @@ type CheckpointSavedProps = {
 export const CheckpointSaved = (props: CheckpointSavedProps) => (
 	<div className="flex items-center justify-between">
 		<div className="flex items-center gap-2">
-			<span className="codicon codicon-git-commit" />
+			<span className="codicon codicon-git-commit text-blue-400" />
```

Comment:
I don't think this plays nicely with VSCode themes. I'll see if their CSS vars have some kind of color we can use to make this stand out.